Servers/UI/OJS.Servers.Ui/ClientApp/src/styles/fonts.scss

```diff
@@ -0,0 +1,29 @@
+﻿$font-family-lato: Lato, serif;
+$font-family-lato-regular: Lato-Regular, serif;
+$font-family-montserrat: Montserrat, serif;
+$font-family-montserrat-medium: Montserrat-Medium, serif;
+$font-style-normal: normal;
+
+@mixin font-family-lato {
+  font-family: $font-family-lato;
+  @include font-style-normal;
+}
+
+@mixin font-family-lato-regular {
+  font-family: $font-family-lato-regular;
+  @include font-style-normal;
+}
+
+@mixin font-family-montserrat {
+  font-family: $font-family-montserrat;
+  @include font-style-normal;
+}
+
+@mixin font-family-montserrat-medium {
+  font-family: $font-family-montserrat-medium;
+  @include font-style-normal;
+}
+
+@mixin font-style-normal {
+  font-style: $font-style-normal;
+}
```

Comment:
Flavor `placeholders` over `mixins`.

Generally, `placeholders` generate **less output css**.

Mixins should be used when parameterized.

Nice short article: https://medium.com/@linker21/scss-placeholders-vs-mixins-fe79b9dbb6d5